ITE/9/README.adoc

```diff
@@ -45,7 +45,26 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification MUST define unambiguously a unique `predicateType`, a shorter
+`predicateName` following the rules defined by the attestation framework, and
+contain a detailed schema of the fields that make up the predicate. The name of
```

Comment:
I wonder if we can be a little more precise and explicit here, still. As in, currently the "front matter" of a predicate spec contains a line for Type URI, which ends up mapping to the `predicateType`, a Version, which in some cases is/isn't included in the Type URI, and now I suppose we're adding a requirement for a `predicateName`, but it's not entirely clear where in the spec that should be indicated. I assumed this front matter section. 

Should we a) explicitly state that the Type URI + Version at the top should map to the `predicateType` in the Statement? (Please correct me if that's already indicated somewhere else). And b) Should we also explicitly mention which section of the spec we expect the predicate name to defined in?